x-pack/plugins/cases/server/client/cases/find.ts

```diff
@@ -101,3 +105,21 @@ export const find = async (
     });
   }
 };
+
+const validateSearchFields = (searchFields?: string | string[]) => {
+  const invalidSearchFields = getUnsearchableFields(uniq(asArray(searchFields)));
+
+  if (invalidSearchFields.length > 0) {
+    throw Boom.badRequest(
+      `received invalid search fields: [${invalidSearchFields.join(',')}], must be one of [${[
+        ...SEARCHABLE_FIELDS,
+      ].join(',')}]`
+    );
+  }
+};
+
+const getUnsearchableFields = (fields: string[]) => {
+  return fields.filter((field) => !SEARCHABLE_FIELDS.has(field));
+};
+
+const SEARCHABLE_FIELDS = new Set<string>(['description', 'connector.name', 'title']);
```

Comment:
@cnasikas @peteharverson do we want to search `connector.name`? I don't think it's actually listed in the cases list page so maybe it'd be confusing to users?

Reply:
I think it does not make that sense to search the connector. Though I can see someone trying to filter all cases based on a specific connector. Let's keep it simple at the moment. What do you think?

Reply:
Sounds good to me! I'll remove it 👍